content/400-reference/200-api-reference/300-environment-variables-reference.mdx

```diff
@@ -92,6 +92,17 @@ This environment variable is available since version `5.2.0`
 
 `PRISMA_GENERATE_NO_ENGINE` can be set to a truthy value to generate a Prisma Client without an included [query engine](/concepts/components/prisma-engines/query-engine) in order to reduce deployed application size when paired with [Prisma Data Proxy](/data-platform/classic-projects/data-proxy) or [Prisma Accelerate](/data-platform/accelerate/).
 
+### <inlinecode>PRISMA_SCHEMA_DISABLE_ADVISORY_LOCK</inlinecode>
+
+<Admonition type="info">
+
+This environment variable is available since version `5.3.0`
+
+</Admonition>
+
+`PRISMA_SCHEMA_DISABLE_ADVISORY_LOCK` can be set to a truthy value to disable the [advisory locking](/concepts/components/prisma-migrate/migrate-development-production#advisory-locking) used by Prisma Migrate. This might be needed, depending on the database configuration for example, for a Percona-XtraDB-Cluster or MariaDB Galera Cluster.
```

Comment:
```suggestion
`PRISMA_SCHEMA_DISABLE_ADVISORY_LOCK` can be set to a truthy value to disable the [advisory locking](/concepts/components/prisma-migrate/migrate-development-production#advisory-locking) used by Prisma Migrate. This might be needed, depending on the database configuration, for example, for a Percona-XtraDB-Cluster or MariaDB Galera Cluster.
```